_posts/nurahill/2017-05-22-nura-tirtle-hack.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+author: nurahill
+title: "Nura's Turtle Exercise"
+---
+
+Here is the program i am embeding:
+ <iframe src="https://trinket.io/embed/python/0af7db0563" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+ 
+ This assignment was fun. Just like the first one. When I started the assignment I had no idea what i wanted to do. So i googled things for inspiration. I did not know that we had to cite where we got code from if we used someone elses code, but i will do my best to find and it add it somewhere. So basically what I designed was a logo (because thats kind of what it looks like). See image below.
```

Comment:
Glad it was fun!

Many (all?) of them are from here: https://michael0x2a.com/blog/turtle-examples